filebeat/crawler/prospector.go

```diff
@@ -28,9 +31,24 @@ type Prospectorer interface {
 	Run()
 }
 
-func NewProspector(prospectorConfig cfg.ProspectorConfig, registrar *Registrar, spoolerChan chan *input.FileEvent) (*Prospector, error) {
+type prospectorConfig struct {
```

Comment:
I added the method but currently left it empty. Config stuff was moved to config.go